autoload/rust.vim

```diff
@@ -514,8 +514,10 @@ function! rust#Test(all, options) abort
         return rust#Run(1, '--test ' . a:options)
     endif
 
-    if has('terminal') || has('nvim')
+    if has('terminal')
         let cmd = 'terminal '
+    elseif has('nvim')
+        let cmd = 'new | terminal '
```

Comment:
```suggestion
        let cmd = 'noautocmd new | terminal '
```

Many plugins run their code on `BufEnter`, `WinEnter` etc. Avoid this using `:noautocmd`.

Reply:
Nice thanks.